app/components/LoginCard/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+*
+* LoginCard
+*
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Well, FormControl, FormGroup, ControlLabel, Button } from 'react-bootstrap';
+
+class LoginCard extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.results = {};
+    this.confirm = null;
+    this.submit = this.submit.bind(this);
+    this.changeForm = this.changeForm.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {
+      toggle: 'Register',
+      current: 'Login',
+    };
+  }
+  changeForm() {
+    if (this.state.toggle === 'Register') {
+      this.setState({ toggle: 'Login', current: 'Register' });
+    } else {
+      this.setState({ toggle: 'Register', current: 'Login' });
+    }
+  }
+  handleChange() {
+    const password = document.getElementById('password');
+    const confirmPassword = document.getElementById('confirmPassword');
```

Comment:
Change these to refs so we aren't searching the DOM each time they click a key.

Reply:
:anger: u r stinky :anger: